meson.build

```diff
@@ -566,12 +566,13 @@ if polkit.found()
   subdir('policy')
 endif
 if build_standalone
+  plugin_quirks = []
   gcab = find_program('gcab', required: get_option('tests'))
   subdir('data')
   subdir('po')
   subdir('libfwupdplugin')
-  subdir('src')
   subdir('plugins')
+  subdir('src')
```

Comment:
why this change?  I think if you're gonna move this around just make the whole list alphabetical

Reply:
the stuff plugins adds the gresource files into the array, then when in src:

 * in the non-gresource case, , and then they get installed into quirks.d
 * in the gresource case, they get built into the ./src/fwupd binary and ignored

Reply:
meson always parses meson.build files in a predictable linear order. It only parallelizes when all the targets are known.

Reply:
Then I think we need to leave a comment here that this order is important

Reply:
done.